scripts/migrations/drop_holds_detected_column.py

```
@@ -0,0 +1,492 @@
+                """
+                connection.execute(text(copy_sql))
+
+                # Recreate indexes
+                for index in indexes:
+                    if index["name"] and index["column_names"]:
+                        cols = ", ".join(index["column_names"])
+                        unique = "UNIQUE" if index["unique"] else ""
+                        index_sql = f"CREATE {unique} INDEX {index['name']} ON {table_name} ({cols})"
+                        try:
+                            connection.execute(text(index_sql))
+                        except Exception as e:
```

Pylint (4.0.4)

[warning] Catching too general exception Exception

(W0718)